source/cpu/security_amd64.go

```diff
@@ -158,3 +151,20 @@ func getCgroupMiscCapacity(resource string) int64 {
 
 	return totalResources
 }
+
+func getCgroupMiscCapacity(resource string) int64 {
+	var totalResources int64 = -1
```

Comment:
You could just drop this and return -1 in the end

```suggestion
```

Reply:
Updated.